### DISCUSSION TITLE
How to backup / restore devices, groups, profiles, users

### DISCUSSION THREAD

**Discussion by r-j-s** [seriousm4x/UpSnap#783] [Help wanted] [answered]
Hello,

thank you for this project.

As you can read in the title I want to backup my device, groups, etc if something bad happens. After this I want to be able to restore all my data as well.
Unfortunately I didn't found any information in the "Wiki" section so I created this discussion.

In detail my question is, can I backup the database from a filesystem directory or is it possible to make a backup of the db tables with the pocketbase dashboard? Can you provide a wiki article where the information is described?

I could imagine this would be helpful for others as well.

Thank you.

Kindest regards

rjs

**Comment by seriousm4x** [comment, marked as answer]
Hi, you can either copy the whole pb_data directory while UpSnap is not running, or use pocketbase's backup and restore system described here: https://pocketbase.io/docs/going-to-production/#backup-and-restore

You can reach the backend by opening http://localhost:8090/_/

It also offers an api for backups: https://pocketbase.io/docs/api-backups/

**Reply by r-j-s** [reply]
Hey seriousm4x,

thank you for the fast response. I will do it like you described.

Have a nice day.
